config/travis.example.yml

```diff
@@ -0,0 +1,176 @@
+
+defaults_sentry: &defaults_sentry
+  dsn: empty
+
+
+#############################################################################################
+development:
+  log_level: :warn
+  origin_whitelist: '*'
+  allowed_origin: '*'
+  log_file_storage_path: <%= ENV['FINAL_OUTPUT_LOGS_DIR'] %>
+  database:
+    <<: *defaults_db
+    database: <%= ENV['FINAL_DATABASE'] %>
+  logs_database:
+    <<: *defaults_db
+    database: <%= ENV['FINAL_LOGS_DATABASE'] %>
+  amqp:
+    <<: *defaults_amqp
+  redis:
+    url: redis://<%= ENV['RE_PORT_6379_TCP_ADDR'] %>:<%= ENV['RE_PORT_6379_TCP_PORT'] %>/1
+#sentry:
+#  <<: *
+
+#specific settings for travis logs
+  logs:
+    archive: true
+    purge: true
+    threads: 10
+    intervals:
+      regular: 3
+      vacuum: 10
+      force: 14400
+      purge: 6
+  log_storage_provider: file_storage
+  file_storage:
+    root_path: <%= ENV['FINAL_OUTPUT_LOGS_DIR'] %>
+
+#specific settings for travis hub
+  queue:
+    limit:
+      default: 500
+    interval: 3
+  #Each job has assigned particular queue[travis-core/lib/travis/model/job/queue.rb] against attributes in queues array.
+  queues:
+    - queue: builds.linux
+      os: linux
+    - queue: builds.windows.static
+      os: windows_static
+    - queue: builds.windows.openstack
+      os: windows2
+    - queue: builds.windows.labmanager
+      os: windows
+    - queue: builds.osx
+      os: osx
+    - queue: builds.example
+      slug:     lksv/best_repo
+      owner:    lksvXX
+      language: rubyXX
+      os:       specialOS
+      sudo:
+      education:
+  encryption:
+    key: your encryption key
+  stash:
+    source_host: stash.yourcompany.com
+    source_port: 443
+    source_protocol: https
+    git_source_protocol: ssh
+    git_source_port: 8085
+    api_url: https://stash.yourcompany.com/rest/api/1.0/
+    oauth:
+      secret: |
+        -----BEGIN RSA PRIVATE KEY-----
+        your oauth RSA key
+        -----END RSA PRIVATE KEY-----
+      key: 'your oauth key'
+    ssl:
+      verify: false
+
+
+
+#############################################################################################
+production:
+  log_level: :warn
+  origin_whitelist: '*'
+  allowed_origin: '*'
+  log_file_storage_path: <%= ENV['FINAL_OUTPUT_LOGS_DIR'] %>
+  database:
+    <<: *defaults_db
+    database: <%= ENV['FINAL_DATABASE'] %>
+  logs_database:
+    <<: *defaults_db
+    database: <%= ENV['FINAL_LOGS_DATABASE'] %>
+  amqp:
+    <<: *defaults_amqp
+  redis:
+    url: redis://<%= ENV['RE_PORT_6379_TCP_ADDR'] %>:<%= ENV['RE_PORT_6379_TCP_PORT'] %>/10
+
+#specific settings for travis hub
+  queue:
+    limit:
+      default: 500
+    interval: 3
+  #Each job has assigned particular queue[travis-core/lib/travis/model/job/queue.rb] against attributes in queues array.
+  queues:
+    - queue: builds.linux
+      os: linux
+    - queue: builds.windows.static
+      os: windows_static
+    - queue: builds.windows.openstack
+      os: windows2
```

Comment:
windows2 for `os` is not ideal, our modification of travis-worker uses `os` to use right wrapper script.